locations/spiders/dats24.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+import scrapy
+
+from locations.items import Feature
+
+
+class Dats24BESpider(scrapy.Spider):
+    name = "dats24_be"
+    item_attributes = {"brand": "DATS 24", "brand_wikidata": "Q15725576"}
+    start_urls = ["https://customer.dats24.be/wps/portal/datscustomer/fr/b2c/locator"]
+
+    def parse(self, response, **kwargs):
+        script_tags = response.text.split("<script")
+        for script_tag in script_tags:
+            if 'class="locatorMapData"' in script_tag:
+                start_index = script_tag.find(">") + 1
+                end_index = script_tag.find("</script>", start_index)
+                json_data = script_tag[start_index:end_index]
+                break
```

Comment:
FYI, this could have been an xpath:

```suggestion
        json_data = response,xpath('//script[@class="locatorMapData"]/text()').get()
```